### DISCUSSION TITLE
No documentation for the "Auto Merge" button?

### DISCUSSION THREAD

**Discussion by johnhaire89** [WinMerge/website#28] [Q&A]
Dear WinMerge team
I can't find any documentation on what the Auto Merge button does.
Perhaps a description could be appended to Section 5.2 of https://manual.winmerge.org/en/Compare_files.html
I might've submitted a pull request but I'm not sure what the button does.

**Comment by sdottaka** [comment]
Thank you for your interest in "Auto Merge".

The source of https://manual.winmerge.org/en/Compare_files.html is as follows
https://github.com/WinMerge/winmerge/blob/master/Docs/Manual/EN/Compare_files.xml

"Auto Merge" is a feature that performs a three-way merge.

WinMerge's 3-way merge is a bit different than other applications, depending on which pane has focus.

 - When you auto-merge in the middle pane, it's assumed that files are listed in the order [Other's file]-[Original file]-[My file]. Changes on the left (other) and right (self) are merged into the middle (original).

- When you auto-merge in the right pane, it's assumed that files are listed in the order [Original File]-[Other's File]-[My File]. The changes in the middle (others) are merged into the right side (yourself).
 
- When you auto-merge in the left pane, it's assumed that files are listed in the order [My File]-[Others' File]-[Original File]. The changes in the middle (others) are merged into the left side (yourself).

Conflicts are left unchanged without the conflict mark.

Conflicts are marked red in the location pane.
